data/campaigns/Winds_of_Fate/scenarios/09_Ancestor.cfg

```diff
@@ -719,7 +719,7 @@ I am elder of Green Swamp Clan and all my years Shek'kahan protected usss. I cou
             speaker=Khrakrahs
             image_pos=right
             mirror=yes
-            message= _ "It was nigh every season he would intrude upon my domain or me his. Each time, there was such a glorious battle between we, the Sons of Drugis, as to impart even the clouds with fright. I must not have challenged him enough that he grew weak, vulnerable. Now another great battle between us there will never be. Now there is only me."
+            message= _ "It was nigh every season he would intrude upon my domain or me his. Each time, there was such a glorious battle between we, as to impart even the clouds with fright. I must not have challenged him enough that he grew weak, vulnerable. Now another great battle between us there will never be. Now there is only me."
```

Comment:
This reads fine with the appositive "Sons of Drugis" but when you remove it, it sounds strange. Shouldn't it just be "us"? Possibly also add "so"?

…between us, so as to impart…